designer/client/src/brace/theme/nussknacker.css

```diff
@@ -10,6 +10,15 @@
     width: 1px;
     background: #555651
 }
+
+.ace-nussknacker .ace_search {
+    background: #818181;
```

Comment:
In the case of edit node fields, we use `#333` color, maybe it's worth it to be consistent with existing fields?

Reply:
I  don't think it is good idea since `#333` is the same as input background.

Reply:
Ohh, I thought that it was the color of a background of the input :P